doc/user-guide/user-guide.rst

```diff
@@ -86,13 +91,27 @@ Local Mode
 You can control the M2 directly from the local mode (click the **Local** button after the connection with the controller is on).
 If the system is under the remote control originally, this action will take over the control, and the M2 cell controller listens to the command from GUI only.
 
+Before the state transition, you need to connect to the M2 cell first.
+Then, you should check the **Enabled Faults Mask** value should not be 0.
+See the :ref:`lsst.ts.m2gui-user_alarm_warn` for the details.
+If it is 0, you will need to restart the application (see :ref:`lsst.ts.m2gui-error_restart_control_system`) to reset the internal data.
+
 You can transition the system into the **Diagonostic** state to check the system status without energizing the actuators.
-If everything looks good, you can transition the system into the **Enabled** state to command the M2 hardware with the open-loop or closed-loop control.
+By default, the **Diagonostic** state does not have the powers of communication and motor.
+It would be good that you can check the interlock first before transitioning to the **Enabled** state.
 
 Based on the circuit configuration, the interlock might be released after the motor power is on or the motor power could not be turned on because the interlock signal is still there.
 You can use the **Diagnostics** table to toggle the motor/communication power to check the circuit of interlock at that time if you could not enable the system successfully.
 See the :ref:`lsst.ts.m2gui-user_diagnostics` for more details.
 
+If everything looks good, you can transition the system into the **Enabled** state to command the M2 hardware with the open-loop or closed-loop control (do not forget to turn off the motor and communication power first if this will be the first time to transition to the **Enabled** state).
+If the interlock is on, you would not be able to transition to the **Enabled** state.
+Sometimes, you may need the global interlock system (GIS) to reset the interlock signal of M2.
```

Comment:
add: By pushing the RESET blue button of the M2 ACTUATOR tab on the GIS cabinet at the level 2

Reply:
Added in the commit: [2fbe0cc](https://github.com/lsst-ts/ts_m2gui/pull/55/commits/2fbe0ccb8c2407391d367a675d2b3a2c410b8fb1). Thanks!